### DISCUSSION TITLE
Splitnn requires NVFlare 2.4

### DISCUSSION THREAD

**Discussion by eshatkeinensinn** [NVIDIA/NVFlare#2003] [Q&A] [answered]
In the requirements.txt of splitnn in advanced/vertical_federated_learning/cifar10_splitnn u need NVFlare 2.4 or higher. Should i use the 2.4.0rc1 or is there just a mistake in the requirements?
lg Jakob

**Comment by YuanTingHsieh** [comment, marked as answer]
@eshatkeinensinn  thanks for the question!
If you want to use the latest version of NVFlare (it is unstable though), you can ignore this requirements and instead do `pip install -e .[dev]` in the cloned NVFlare repo.

If you want to use the stable release version of NVFlare (2.3) you should use examples from 2.3 branch: https://github.com/NVIDIA/NVFlare/tree/2.3/examples/advanced/vertical_federated_learning
